Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -4790,9 +4790,14 @@ bool CStaticFunctionDefinitions::GetCameraMatrix(CVector& vecPosition, CVector&
 
 CClientEntity* CStaticFunctionDefinitions::GetCameraTarget()
 {
-    if (!m_pCamera->IsInFixedMode())
-        return m_pCamera->GetTargetEntity();
-    return NULL;
+    if (m_pCamera->IsInFixedMode())
+        return NULL;
+
+    CClientEntity* pPlayer = m_pCamera->GetFocusedPlayer();
+    if (pPlayer)
+        return pPlayer;
```

Comment:
```cpp
    if (CClientEntity* pPlayer = m_pCamera->GetFocusedPlayer())
        return pPlayer;
```

Reply:
It's a good practice to limit the scope for variables, but this is optional. It's like one of those changes where you might want to use braces for a single line after if condition (I don't use braces in that case). In such cases, I prefer to let the contributor decide how they want to approach it.

Reply:
I think first way is easy to read